api/IDBDatabase.json

```
@@ -98,6 +98,54 @@
           }
         }
       },
+      "abort_event": {
+        "__compat": {
+          "description": "`abort` event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/IDBTransaction/abort_event",
+          "spec_url": [
+            "https://w3c.github.io/IndexedDB/#transaction-abort",
+            "https://w3c.github.io/IndexedDB/#dom-idbdatabase-onabort"
+          ],
+          "tags": [
+            "web-features:indexeddb"
+          ],
+          "support": {
+            "chrome": {
+              "version_added": "23"
+            },
+            "chrome_android": "mirror",
+            "edge": {
+              "version_added": "12"
+            },
+            "firefox": {
+              "version_added": "10"
+            },
+            "firefox_android": {
+              "version_added": "22"
+            },
+            "ie": {
+              "version_added": "10",
+              "partial_implementation": true,
+              "notes": "Unknown limitations"
```

Probably would make more sense just to get rid of `partial_implementation` if we don't know what the limitations are.

```suggestion
              "version_added": "10"
```

---

Since IE (intentionally) diverged from the spec (and never implemented the 2.0 spec), I think it's very likely that this event _does_ have weird or incomplete behavior (see https://github.com/Fyrd/caniuse/pull/532).

Instead of sweeping the data quality issues here under the rug, I'd suggest making it explicit:

```suggestion
              "version_added": "10",
              "partial_implementation": true,
              "notes": "Internet Explorer's IndexedDB implementation is known to be incomplete. The specific behavior of this event is poorly understood, but historic data suggests it has limitations. If you have more information about non-standard behaviors for this event, [file an issue or PR](https://github.com/mdn/browser-compat-data)."
```

---

I would suggest to just leave it as is. Between removing the note altogether (including the partial implementation) and the verbose note, the current note is a good enough compromise.

---

The status quo is fine with me.